internal/pkg/keys/encrypter_init.go

```diff
@@ -0,0 +1,147 @@
+package keys
+
+import (
+	"fmt"
+	"github.com/frain-dev/convoy/database"
+	"github.com/frain-dev/convoy/pkg/log"
+	"github.com/jmoiron/sqlx"
+	"strings"
+)
+
+func InitEncryption(db database.Database, km KeyManager, encryptionKey string) error {
+	// Start a transaction
+	tx, err := db.GetDB().Beginx()
+	if err != nil {
+		return fmt.Errorf("failed to begin transaction: %w", err)
+	}
+
+	for table, columns := range tablesAndColumns {
+		log.Infof("Processing table: %s", table)
+
+		if err := lockTable(tx, table); err != nil {
+			_ = tx.Rollback()
```

Comment:
We should handle the error from the transaction rollback or at least log it out. There's a function we created it's called `rollbackTx()` I think

Reply:
OK. Will check